### PR TITLE
Create Github Action

### PR DESCRIPTION
**Pull request by conor081nash**
Initial attempt at a git hub action

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+    name: Push to main action
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
```
Comment (jamieyoung5) [inline .github/workflows/docker-image.yml:20]: java version 11 but we are using 18!!!!!!

**Review by jamieyoung5 — CHANGES_REQUESTED** [review]

**Review by breynolds-nable — COMMENTED** [review]
Thanks Connor, big effort on this request and you came through, customer was super happy. Shame on the Java version issue, lets pick on on this before approval.

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
```
Comment (jamieyoung5) [inline .github/workflows/docker-image.yml:5]: change to develop

**Review by jamieyoung5 — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          java-version: '18'
+          distribution: 'adopt'
+      - name: Compile with Maven
+        run: mvn compile
+      - name: Build Docker Image
+        run: docker build -t semimage .
+      - name: Run image
+        run: docker run --name semcontainer -d semimage
+      - name: view logs
+        run: docker logs semcontainer
```
Comment (breynolds-nable) [inline .github/workflows/docker-image.yml:29]: Change this to use our dockerfile: 

docker build -t dockerfile .
docker run --rm dockerfile
docker logs dockerfile

**Pull request closed without merging**